### PR TITLE
File added: Assignment 5 and 6

### PR DESCRIPTION
**Pull request by shivani-pawbake**

### CODE REVIEW

File: Shivani/HTML/assign6/first.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="code1">
+            <h1>MI-11 Intern HTML Assesment</h1>
+        </div>
+        <hr />
+        <div class="code2">
+            <h4>Personal Details</h4>
+        </div>
+        <form class="a">
+            <label for="fname"><b>First name:</b></label>
+            <input type="text" id="fname" name="fname" /><br /><br />
```
Comment (pooja-nagare) [inline Shivani/HTML/assign6/first.html]: add value or placeholder for First Name

File: Shivani/HTML/assign6/first.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <label for="lname"><b>Last name:</b></label>
+            <input type="text" id="lname" name="lname" /><br /><br />
```
Comment (pooja-nagare) [inline Shivani/HTML/assign6/first.html]: add value or placeholder for Last Name

File: Shivani/HTML/assign5/first.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            </p>
+            <h4>More About me</h4>
+            <ul class="p">
+                <li><a href="a" target="_blank">My Resume</a></li>
+                <li><a href="b" target="_blank">My Hobbies</a></li>
+                <li><a href="c" target="_blank">Photo Gallery</a></li>
+            </ul>
+        </div>
+    </body>
+</html>
```
Comment (shivnanda6272) [inline Shivani/HTML/assign5/first.html:26]: add new line.

File: Shivani/HTML/assign6/first.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            <label for="gender"><b>Gender:</b></label>
+            <input type="radio" id="male" name="gender" value="male" />
+            <label for="male"><b>Male</b></label>
+            <input type="radio" id="female" name="gender" value="female" />
+            <label for="female"><b>Female</b></label>
+            <input type="radio" id="other" name="gender" value="other" />
+            <label for="other"><b>Other</b></label
+            ><br /><br />
```
Comment (shivnanda6272) [inline Shivani/HTML/assign6/first.html]: correct the indentation

File: Shivani/HTML/assign6/first.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            <label for="html"><b>HTML</b></label>
+            <input type="checkbox" id="Java script" name="Javascript" />
+            <label for="java script"><b>Java script</b></label>
+            <input type="checkbox" id="Java" name="skill set" />
+            <label for="java"><b>Java</b></label>
+            <input type="checkbox" id="Oracle" name="skill set" />
+            <label for="oracle"><b>Oracle</b></label>
+        </form>
+    </body>
+</html>
```
Comment (shivnanda6272) [inline Shivani/HTML/assign6/first.html:54]: add extra line

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
do changes.

**Review by pooja-nagare — APPROVED** [review]

File: Shivani/HTML/assign6/first.html [hunk 6]
```diff
@@ -40,7 +40,7 @@ <h4>Personal Details</h4>
             <h4>Academic Details</h4>
         </div>
         <form class="a">
-            <label for="skill set"><b>Skill Set:</b></label>
+            <label for="skill set"></label><b>Skill Set:</b></label>
```
Comment (shivnanda6272) [inline Shivani/HTML/assign6/first.html]: why you are closed <label >tag  two time . do it proper...

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
make changes.

**Review by shivnanda6272 — APPROVED** [review]

**Review by pritikekan — APPROVED** [review]

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**